droidlet/interpreter/robot/tasks.py

```diff
@@ -558,7 +568,7 @@ def step(self):
             logging.info(f"Current Pos {base_pos}")
             logging.info(f"Move Target for Examining {tloc}")
             logging.info(f"Distance being moved {np.linalg.norm(base_pos[:2]-tloc[:2])}")
-            self.add_child_task(Move(self.agent, {"target": tloc}))
+            self.add_child_task(Move(self.agent, {"target": tloc, "is_blocking": True}))
```

Comment:
the is_blocking is to stop some other agent behavior from taking over the explore?  what was happening?

Reply:
This is blocking so that reexplore only saves the frames that is on a waypoint (or its best estimate). For example if the hello maybe swivels around randomly to go somewhere, this will ensure we disregard those intermediate frames.